src/wad/level.rs

```diff
@@ -152,12 +150,14 @@ impl Level {
         let mut min_light = sector.light;
         let sector_id = self.sector_id(sector);
         for line in self.linedefs.iter() {
-            let (left, right) =
-                (match self.left_sidedef(line) {
-                    Some(l) => l.sector, None => continue,
-                 }, match self.right_sidedef(line) {
-                    Some(r) => r.sector, None => continue,
-                 });
+            let (left, right) = (match self.left_sidedef(line) {
+                Some(l) => l.sector,
```

Comment:
That makes sense in general, but I think I'd only do that if the match arms are the last argument in a call or last element in a tuple. If it's in the middle it looks a bit dodgy.